### PR TITLE
Py3 Runtime: distribute the tests

### PR DESCRIPTION
**Pull request by mr-c**
Hello,

I'm packaging the Python 3 runtime for Debian. It would be useful to us if the source distribution (on https://pypi.org/project/antlr4-python3-runtime/ or elsewhere) contained the tests.

I've modified it so that the tests are now part of the installed Python module; with this PR users can confirm their installation with `python3 -m unittest antlr4.test`

Many thanks,

### CODE REVIEW

**Comment by mr-c** [comment]
I just learned that pulling the tests out of the sdist was a recent happening: https://github.com/antlr/antlr4/pull/2911

Still, Debian would appreciate having the tests in the sdist.

**Comment by ericvergnaud** [comment]
there are hardly any tests in the python runtime itself
the real tests are part of antlr4-complete, a java package
why do you need the tests?

**Comment by mr-c** [comment]
@ericvergnaud We'd like to confirm that the runtime works when we build the Debian package, and that it the produced Debian package continues to work even if there are changes to our Python 3 package or other dependencies.

**Comment by mr-c** [comment]
If there are tests of the Python3 runtime available only on the Java side, then I can co-ordinate with the Debian volunteers who maintain https://tracker.debian.org/pkg/antlr4 ; but we would still appreciate making use of any pure-Python dedicated tests that exist for the Python3 runtime.

**Comment by mr-c** [comment]
@ericvergnaud I've reworked this PR to only include the `test` directory in the sourceball, without including the tests in the `antlr4` Python package. I think this approach avoids the issue the #2911 fixed while still distributing the tests when the next update is published to PyPI.

As a bonus, I've retained the change that includes the `pygrun` script as part of the installation process.

Please let me know if this is more acceptable, thanks!

File: runtime/Python3/MANIFEST.in [hunk 1]
```diff
@@ -1 +1,2 @@
-include *.txt
\ No newline at end of file
+include *.txt
+recursive-include test *.py *.c
```
Comment (ericvergnaud) [inline runtime/Python3/MANIFEST.in:2]: why *.c ?

Comment (mr-c) [reply runtime/Python3/MANIFEST.in:2]:
```
~/src/antlr4/runtime/Python3$ git ls-files | grep '\.c$'
test/c.c
```

Comment (ericvergnaud) [reply runtime/Python3/MANIFEST.in:2]: I see, thanks

**Comment by mr-c** [comment]
Thanks!

**Pull request merged**